### PR TITLE
docs: Add macOS settings to getting started

### PR DESCRIPTION
**Pull request by ansabgillani**
Add instructions to run agoric app on macOS

### CODE REVIEW

File: main/guides/getting-started/index.md [hunk 1]
```diff
@@ -39,20 +39,49 @@ In this section you'll be installing prerequisite components into your environme
...
+Then, you can install nvm using Brew
+```sh
+brew install nvm
+```
+
+Add nvm to your shell profile: To make nvm available every time you open a new terminal window, you will need to add the following line to your shell profile (e.g., ~/.bash_profile or ~/.zshrc):
+
+```sh
+source $(brew --prefix nvm)/nvm.sh
+```
```
Comment (gibson042) [inline main/guides/getting-started/index.md:81]: Do the instructions at https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating not work on Mac?

Comment (ansabgillani) [reply main/guides/getting-started/index.md:81]: Yes, the `curl` command works. But brew is more native to installing the libraries in Mac systems. Especially when installing Docker.

**Review by toliaqat — APPROVED** [review]

File: main/guides/getting-started/index.md (hunk 1 above)
Comment (mudassir-agoric) [inline main/guides/getting-started/index.md:81]: I think homebrew is installed on every developer's mac - so instead just mention the command to install `nvm` using `brew` and then say that 'you will need to install `homebrew` using instructions at the following link' and provide link that.

File: main/guides/getting-started/index.md [hunk 2]
```diff
@@ -74,6 +102,13 @@ yarn --version # for verification
 
 ### Installing Docker
 
+<details>
+<summary>Installing Docker</summary>
+
+For docker, you can either download Docker UI using the steps on the [installation for Ubuntu](https://docs.docker.com/desktop/install/ubuntu/) or [installation for Ubuntu](https://docs.docker.com/desktop/install/mac-install/)
```
Comment (mudassir-agoric) [inline main/guides/getting-started/index.md:108]: the second link should be 'installation for Mac'. OR you can simply provide link for both at https://docs.docker.com/get-docker/

File: main/guides/getting-started/index.md [hunk 3]
```diff
@@ -101,6 +136,18 @@ Now you can install Docker!
 sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
 ```
 
+</details>
+
+<details>
+<summary>Installing Docker on MacOS</summary>
+Previously, you installed brew on your machine. You can install docker using the same command.
```
Comment (mudassir-agoric) [inline main/guides/getting-started/index.md:143]: Remove this: Previously, you installed brew on your machine.

**Pull request merged**